aldryn_reversion/admin.py

```diff
@@ -116,39 +117,52 @@ def _get_placeholder_attached_object(self, placeholder):
             obj = None
         return obj
 
+    def post_clear_placeholder(self, request, placeholder):
+        comment_dict = {'placeholder': placeholder}
+        comment = _(u'All plugins in the placeholder '
+                    u'"%(placeholder)s" were deleted.') % comment_dict
```

Comment:
Why ``u''` if we have unicode literals from future imported? you can omit the ``u`` since it would be unicode anyway.
And why not ''.format? (optional)